packages/flame_test/lib/src/flame_test.dart

```diff
@@ -81,6 +81,9 @@ class GameTester<T extends Game> {
 
     await game.onLoad();
     game.update(0);
+    if (game is FlameGame) {
```

Comment:
I am not sure if we should hard code this, there may be situations where the user will want to do assertions before the game is ready and after.

Reply:
`await game.ready()` serves the same purpose as `game.update(0)`, only it's more reliable: the queues are flushed not once but repeatedly until they're empty. Thus, this line improves upon the existing behavior without fundamentally altering it.

> there may be situations where the user will want to do assertions before the game is ready and after.

I agree, this could be useful in limited number of circumstances, such as for example when testing the `FlameGame` class itself. But in those tests we would probably want to avoid using `GameTester` and invoke all relevant lifecycle methods manually.

Reply:
Still I would prefer to give the user the option. I think that an optional book argument, defaulting to true would gives us the best of both worlds

Reply:
Is `book` the name of the argument?
Also, should it guard only `game.ready` or also `game.update`, `game.onLoad`, `game.onMount` and `game.onGameResize` ?

Reply:
Book was a typo, it was supposed to be `bool`. I think it could guard only the ready, as it is the one being introduced here

Reply:
Ah I see, I was at a loss what did "book" mean :) 
Anyways, added the parameter `makeReady` now, which is true by default but can be set to false if the user so desires.